src/enums.h

```diff
@@ -59,6 +59,8 @@ enum itemAttrTypes : uint32_t {
 	ITEM_ATTRIBUTE_CHARGES = 1 << 20,
 	ITEM_ATTRIBUTE_FLUIDTYPE = 1 << 21,
 	ITEM_ATTRIBUTE_DOORID = 1 << 22,
+
+	ITEM_ATTRIBUTE_CUSTOM = 1 << 30
```

Comment:
Yes I've tried that and got this:
`c:\users\user\documents\github\fork\forgottenserver\src\enums.h(64): warning C4309: 'initializing': truncation of constant value` 
`c:\users\user\documents\github\fork\forgottenserver\src\enums.h(64): warning C4369: 'ITEM_ATTRIBUTE_CUSTOM':  enumerator value '-2147483648' cannot be represented as 'unsigned int', value is '2147483648' `